src/event-detail/ui/event-detail.controller.js

```diff
@@ -23,7 +23,8 @@ function EventDetail(
   $uibModal,
   $q,
   $window,
-  offerLabeller
+  offerLabeller,
+  $timeout
```

Comment:
Hier injecteer je ook $timeout en dat wordt vervolgens ook nergens in de controller gebruikt.